Assets/Scripts/UI/Menu's/GoBackToMainMenu.cs

```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+
+public sealed class GoBackToMainMenu : MenuController
```

Comment:
```suggestion
public sealed class QuitCredits : MenuController
```
De naam kan beter, het voelt nu onnodig lang.